Configuration/StandardSequences/python/GeometryConf.py

```diff
@@ -9,7 +9,7 @@
     'DBExtended2017'   : 'DB:Extended',
     'DBExtended2017ZeroMaterial'    : 'DB:Extended2017ZeroMaterial',
     'DBExtended2018'    : 'DB:Extended',
-    'DBExtended2019'    : 'DB:Extended2019',
+    'DBExtended2021'    : 'DB:Extended2021',
```

Comment:
@boudoul - this should be removed, because we should have a GT for 2021 now where geometry has a default label - "Extended"

Reply:
I can remove this , but in any case in the workflow is already configured to use DB:Extended

Reply:
yes, please. 
@cvuosalo - it would be nice to remove all other irrelevant scenarios from this configuration.